client/src/Components/local/atoms/core-mapping/index.tsx

```diff
@@ -195,8 +195,8 @@ const CoreMapping: React.FC<PropTypes> = ({ messages, channel, playerForce, play
             _type: RENDERER_MILSYM,
             sidc: 'SFG-UCI----D',
             size: 'M',
-            lat: loc.lat,
-            lng: loc.lng,
+            lat: mockLat, // mock lat
```

Comment:
@lebaphi - I think it's _unhealthy_ to store the values as properties - it just feels like it will cause trouble downstream.  As a workaround, could we inject them as special properties before we call `setSelectedProps` in `mapping-panel.tsx`?  We would then extract them again (if present) in `onPropertiesChange`.

Reply:
I don't quite understand your idea, to show the Lat/Lng editor in the properties panel, it has to be defined in `properties`, and it should show the coordinates of the marker?

![image](https://github.com/serge-web/serge/assets/26705621/457147c0-1631-4750-95cd-82e32e25dbae)

Reply:
Sure.  I don't like to see the data stored in 2 places (coordinates and properties).  I can see that's the only way to expose it in the properties panel.  So, instead of it being permanently stored in properties, I'd like us to only have it in there when the properties are being used to build the properties window.  So, immediately before we call `setSelectedProps()` we would check if this is a `point` feature, and insert lat/long properties into the array.  Similarly, on `onPropertiesChange` we would update the `coordinates`, remove the lat/long properties, and pass on the reduced set of properties.

That's all quite complex/convoluted.  I _guess_ an alternate strategy could be for the `PropertiesPanel` to have an optional LatLng value.  If this is provided, introduce a `LatLng` as a first property in the UI.  The `onPropertiesChange` would gain an optional latLng field, populated if there is a LatLng present.